### PR TITLE
Updated en-replacee.de.pairs with translations

### PR DESCRIPTION
**Pull request by Squirkyy**
Added the missing translations.
Also, I think that the level titles should be kept in English

### CODE REVIEW

File: translated-pairs/en-replace.de.pairs [hunk 1]
```diff
@@ -342,48 +342,48 @@ Am Besten
...
-time, run
+time, Durchlauf
 
 solution, run
-solution, run
+Lösung, Durchlauf
 
 time
-time
+Lösung
```
Comment (alexheretic) [inline translated-pairs/en-replace.de.pairs]: shouldn't this be "zeit"?

File: translated-pairs/en-replace.de.pairs [hunk 2]
```diff
@@ -342,48 +342,48 @@ Am Besten
...
-time, solution, run
+Zeit, Lösung, Durchlauf
 
 time, solution
-time, solution
+Zeit, Lösung
 
 time, run
-time, run
+time, Durchlauf
```
Comment (alexheretic) [inline translated-pairs/en-replace.de.pairs]: this "time" is untranslated

**Comment by alexheretic** [comment]
Thanks for the update! 👍 I have just a couple of little observations to clear up.

File: translated-pairs/en-replace.de.pairs (hunk 1 above)
Comment (Squirkyy) [reply translated-pairs/en-replace.de.pairs]: yep, you're right. It's supposed to be "Zeit"

File: translated-pairs/en-replace.de.pairs (hunk 2 above)
Comment (Squirkyy) [reply translated-pairs/en-replace.de.pairs]: Damn two mistakes. Sorry 'bout that. Yeah, that's "Zeit, Durchlauf"

**Pull request merged**